packages/expo-modules-core/ios/ReactDelegates/ExpoReactDelegateHandler.swift

```diff
@@ -35,7 +35,7 @@ open class ExpoReactDelegateHandler: NSObject {
    Otherwise return nil.
    */
   @objc
-  open func createRootViewController(reactDelegate: ExpoReactDelegate) -> UIViewController? {
+  open func createRootViewController() -> UIViewController? {
```

Comment:
fwiw, the original design to pass ExpoReactDelegate for each handlers is for callsite to get correct setup. e.g. in updates when we recreate the root view, we have to use ExpoReactDelegate to get correct root view controller from expo-splash-screen: https://github.com/expo/expo/blob/c3a83979d5e1103d3d259ea537823a3ec3972fa6/packages/expo-updates/ios/EXUpdates/ReactDelegateHandler/ExpoUpdatesReactDelegateHandler.swift#L89.

though the case is not being used here for the `createRootViewController` handler. just share the insight and open to you for making the decision.